Cargo.toml

```diff
@@ -109,6 +109,8 @@ zip = { version = "0.6", default-features = false }
 zstd = "0.13"
 
 # dist-server only
+env_filter = "0.1.3"
```

Comment:
if it is only used for dist, we should make it optional no ?
 (or default-features = false